roles/network_plugin/calico/templates/calico-typha.yml.j2

```diff
@@ -37,6 +37,9 @@ spec:
   # production, we recommend running at least 3 replicas to reduce the impact of rolling upgrade.
   replicas: {{ typha_replicas }}
   revisionHistoryLimit: 2
+  selector:
```

Comment:
maybe add `priorityClassName: system-cluster-critical` and probably other stuff from most recent https://docs.projectcalico.org/v3.9/manifests/calico-typha.yaml ?

Reply:
good suggestion!